tls/s2n_server_new_session_ticket.c

```diff
@@ -56,7 +56,6 @@ int s2n_server_nst_recv(struct s2n_connection *conn) {
 
         if (conn->config->session_ticket_cb != NULL) {
             size_t session_len = s2n_connection_get_session_length(conn);
-            POSIX_ENSURE_GTE(S2N_TLS12_SESSION_SIZE, session_len);
```

Comment:
Why remove this? Is TLS1.2 receiving unusual tickets?

Reply:
s_server tickets are larger than ours in TLS1.2 so this check doesn't make any sense.